src/org/rascalmpl/repl/BaseREPL2.java

```diff
@@ -19,6 +19,7 @@
 import org.jline.terminal.Terminal.Signal;
 import org.jline.terminal.Terminal.SignalHandler;
 import org.jline.utils.ShutdownHooks;
+import org.jline.utils.InfoCmp.Capability;
```

Comment:
New import but no code that uses it?

Reply:
yup, already cleaned, also, this PR is very much WIP ;)